src/System.Resources.ResourceManager/src/System.Resources.ResourceManager.csproj

```diff
@@ -43,6 +43,7 @@
       <Link>Common\System\SR.Core.cs</Link>
     </Compile>
     <TargetingPackReference Include="System.Private.CoreLib.WinRTInterop" />
```

Comment:
No, IsAppXModel is tricky because this is implemented in by overriding the code in the toolchain. we can re-implement it here but we did that before and there was issue that WinRT was loading extra DLL. we can track this issue and do it later if we need to.